axon-server-connector/src/main/java/org/axonframework/axonserver/connector/event/axon/AxonServerEventStore.java

```diff
@@ -481,7 +481,10 @@ public void onCompleted() {
                     requestStream,
                     configuration.getClientId(),
                     configuration.getEventFlowControl(),
-                    t -> GetEventsRequest.newBuilder().setNumberOfPermits(t.getPermits()).build(),
+                    t -> GetEventsRequest.newBuilder()
+                                         .setAllowReadingFromFollower(configuration.isAllowReadingEventsFromFollower())
```

Comment:
Would it make sense to set this field once for the `AxonIQEventStorageEngine` to be used on every `GetEventsRequest` and `QueryEventsRequest` call instead of retrieving it from the config each time? :thinking: